App.tsx

```diff
@@ -75,6 +78,12 @@ const RootStackProvider = () => {
     'darkMode.base'
   );
   const backgroundColorHex = useToken('colors', backgroundColorKey);
+  const setHideSpoilers = useSetRecoilState(hideSpoilersAtom);
+  useEffect(() => {
+    AsyncStorage.getItem('@hideSpoilers').then((val) => {
+      if (val !== null) setHideSpoilers(val === 'true');
+    });
+  }, [setHideSpoilers]);
```

Comment:
Nice! I like the idea of pulling the async storage stuff into global state for ease of use. For the sake of not bloating `App.tsx` any further now or later when we add more settings, can we add this initialization into a new component like `<InitSettings />`, and then include that component in the main tree?

```ts
<RecoilRoot>
  <InitSettings />
   ...
</RecoilRoot>
```